src/Startup/Startup_Leon3.c

```diff
@@ -0,0 +1,38 @@
+/**@file
+ * This file is part of the Leon3 BSP for the Test Environment.
+ *
+ * @copyright 2022-2023 N7 Space Sp. z o.o.
+ *
+ * Test Environment was developed under a programme of,
+ * and funded by, the European Space Agency (the "ESA").
+ *
+ *
+ * Licensed under the ESA Public License (ESA-PL) Permissive,
+ * Version 2.3 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://essr.esa.int/license/list
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/// \brief Startup task definition.
+
+#include "Startup_Leon3.h"
+#include "SystemConfig.h"
+#include <rtems.h>
+#include <rtems/confdefs.h>
+
+extern int main();
+
+rtems_task
+Init(rtems_task_argument arg)
+{
```

Comment:
This is also rather a main wrapper, for use in a demo, not necessarily in a production application

Reply:
OK... but what is the purpose of the rest of the file? Is it even compiled now?